web-client/src/views/FileHandlingHelpers/pdfValidation.ts

```diff
@@ -21,6 +29,20 @@ export const validatePdf = ({
 }: {
   file: File;
 }): Promise<FileValidationResponse> => {
+  if (!clientSupportsES2022()) {
```

Comment:
I do not think we should show this error _until_ the pdf process itself fails. As is, this says, "Don't even try to do the thing, it's an old browser." I think a better user experience is, "Try to do the thing, but if it fails, check if it is an old browser and message accordingly." Does that make sense? But I could be missing something.

Reply:
Also, is it worth caching `clientSupportsES2022`? (Not sure! Just wondering.)

Reply:
I think I need to understand this better. The current code, while it prevents Safari, could also prevent any other browsers that might happen to work.